### ISSUE TITLE
Docstring inheritance v2.1.0 gives multiple `DocstringInheritance` warnings.

### ISSUE THREAD

**Issue by wpk-nist-gov** [AntoineD/docstring-inheritance#12]
Running, for example, `tox -e py311` leads to the following warnings (truncated for simplicity)

```bash
$ tox -e py311
...

============================================= warnings summary ==============================================
tests/test_base_inheritor.py: 10 warnings
  /Users/wpk/gits/docstring-inheritance/.tox/py311/lib/python3.11/site-packages/docstring_inheritance/docstring_inheritors/bases/inheritor.py:329: DocstringInheritanceWarning: File /Users/wpk/gits/docstring-inheritance/tests/test_base_inheritor.py:48: in func_args: section DummyArgs: the docstring for the argument 'arg' is missing.
    self._warn(

tests/test_base_inheritor.py::test_inherit_section_items_with_args[func_args_kwonlyargs-section_items3-expected3]
tests/test_base_inheritor.py::test_inherit_section_items_with_args[func_args_kwonlyargs-section_items4-expected4]
  /Users/wpk/gits/docstring-inheritance/.tox/py311/lib/python3.11/site-packages/docstring_inheritance/docstring_inheritors/bases/inheritor.py:329: DocstringInheritanceWarning: File /Users/wpk/gits/docstring-inheritance/tests/test_base_inheritor.py:52: in func_args_kwonlyargs: section : the docstring for the argument 'arg1' is missing.
    self._warn(

tests/test_base_inheritor.py::test_inherit_section_items_with_args[func_args_kwonlyargs-section_items3-expected3]
tests/test_base_inheritor.py::test_inherit_section_items_with_args[func_args_kwonlyargs-section_items4-expected4]
  /Users/wpk/gits/docstring-inheritance/.tox/py311/lib/python3.11/site-packages/docstring_inheritance/docstring_inheritors/bases/inheritor.py:329: DocstringInheritanceWarning: File /Users/wpk/gits/docstring-inheritance/tests/test_base_inheritor.py:52: in func_args_kwonlyargs: section : the docstring for the argument 'arg2' is missing.
    self._warn(

...
```

All tests still pass, but the warnings are alarming from pytest/sphinx/etc.

**Comment by AntoineD** [comment]
Yes, those warnings are expected.
I'll silence them.

**Comment by AntoineD** [comment]
This has been fixed in version 2.1.1.

**Comment by wpk-nist-gov** [comment]
Thank you!  I'm sorry I wasn't able to save you some work and put together a PR for this.  Hopefully next time!
